dist/images/ovnkube.sh

```diff
@@ -1134,9 +1146,9 @@ ovn-node() {
     --metrics-bind-address ${ovnkube_node_metrics_bind_address} \
      ${ovnkube_node_mode_flag} \
     ${egress_interface} \
-    --host-network-namespace ${ovn_host_network_namespace} &
+    --host-network-namespace ${ovn_host_network_namespace} \
+     ${ovnkube_node_mgmt_port_netdev_flag} &
 
-  wait_for_event attempts=3 process_ready ovnkube
```

Comment:
@adrianchiris Try adding this line back in. I pulled your latest version of the PR to my server and my KIND cluster wouldn't come up. I was seeing locally:
```
kubectl logs -n ovn-kubernetes ovnkube-node-495pf -c ovnkube-node
:
=============== ovn-node ========== running
cat: /var/run/ovn/ovnkube.pid: No such file or directory
```
I also saw that in some of your CI failures. Adding back allowed mine to come up.

Reply:
Thanks billy will do !! 
I was trying to reproduce this locally with kind but for some reason on my setup it worked.

This change must have crept in in the last rebase.